artscraper/artist_links.py

```diff
@@ -0,0 +1,38 @@
+from selenium import webdriver
+from googleart import *
+import time
+
+def get_artist_links(webpage, output_file):
```

Comment:
Instead of an output file, I think it would be nicer just to return a list of links, then the user can decide whether to write it to a file, or simply use it directly.